pkg/codegen/internal/test/sdk_driver.go

```diff
@@ -29,14 +29,68 @@ type sdkTest struct {
 	// are of the form "$language/$check" such as "go/compile".
 	Checks map[string]CodegenCheck
 
-	// Skip checks, identified by "$language/$check".
-	// "$language/any" is special, skipping generating the
+	// KnownBroken indicates tests that are known to be failing, identified by
+	// "$language/$check". "$language/any" is special, skipping generating the
 	// code as well as all tests.
-	Skip codegen.StringSet
+	KnownBroken codegen.StringSet
```

Comment:
I think Skip was maybe a better term than KnownBroken. We're skipping the tests for the other languages in this PR just because they're not useful rather than because there's anything broken with them.